engine/src/main/db/update/scriptPort.sql

```diff
@@ -25,15 +25,15 @@ CREATE TABLE pages_metadata_draft
   titles text NOT NULL,
   showinmenu smallint NOT NULL,
   extraconfig text,
-  updatedat timestamp without time zone,
+  updatedat timestamp without time zone NOT NULL,
   CONSTRAINT pages_metadata_draft_pkey PRIMARY KEY (code),
   CONSTRAINT pages_metadata_draft_code_fkey FOREIGN KEY (code)
       REFERENCES pages (code) MATCH SIMPLE
       ON UPDATE NO ACTION ON DELETE NO ACTION
 );
 
-insert into pages_metadata_draft select code, modelcode, titles,showinmenu, extraconfig from pages;
-insert into pages_metadata_online select code, modelcode, titles,showinmenu, extraconfig from pages;
+insert into pages_metadata_draft select code, modelcode, titles,showinmenu, extraconfig, current_timestamp from pages;
+insert into pages_metadata_online select code, modelcode, titles,showinmenu, extraconfig, current_timestamp from pages;
```

Comment:
in questa maniera tuttavia non finiamo con un valore falso all'interno della tabella?
Io mi aspetterei fossi un cliente di entando 4.2 che quando upgrado alla 4.3 nella tabella della dashboard vedo le reali date di last update. Questo codice invaliderebbe tale risultato.

Abbiamo già un task comunque per creare un vero e proprio script ti migrazione dall'activity stream table per popolare correttamente la tabella.